src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -40,304 +40,187 @@
- * IllegalToken
+ * ImportOrder
  * </li>
  * <li>
- * IllegalTokenText
+ * Indentation
  * </li>
  * <li>
- * IllegalType
+ * InterfaceIsType
  * </li>
  * <li>
- * InnerAssignment
+ * InterfaceMemberImpliedModifier
  * </li>
  * <li>
- * InnerTypeLast
+ * InvalidJavadocPosition
  * </li>
  * <li>
- * InterfaceTypeParameterName
+ * JavadocContentLocation
  * </li>
  * <li>
- * JavadocVariable
+ * JavadocMethod
  * </li>
  * <li>
- * JavaNCSS
+ * JavadocStyle
  * </li>
  * <li>
- * IllegalImport
+ * JavadocType
  * </li>
  * <li>
- * IllegalThrows
+ * LambdaParameterName
  * </li>
  * <li>
- * ImportControl
+ * MethodCount
  * </li>
  * <li>
- * LeftCurly
+ * MissingCtor
  * </li>
  * <li>
- * LocalFinalVariableName
+ * MissingJavadocMethod
  * </li>
  * <li>
- * LocalVariableName
+ * MissingJavadocPackage
  * </li>
  * <li>
- * MagicNumber
+ * MissingJavadocType
  * </li>
  * <li>
- * MemberName
+ * MissingOverride
  * </li>
  * <li>
- * MethodLength
+ * MissingSwitchDefault
  * </li>
  * <li>
- * MethodName
+ * NeedBraces
  * </li>
  * <li>
- * MethodParamPad
+ * NoClone
  * </li>
  * <li>
- * MethodTypeParameterName
+ * NoFinalizer
  * </li>
  * <li>
- * ModifiedControlVariable
+ * NoLineWrap
  * </li>
  * <li>
- * ModifierOrder
+ * OneTopLevelClass
  * </li>
  * <li>
- * MultipleStringLiterals
+ * OuterTypeFilename
  * </li>
  * <li>
- * MultipleVariableDeclarations
+ * OverloadMethodsDeclarationOrder
  * </li>
  * <li>
- * MutableException
+ * PackageAnnotation
  * </li>
  * <li>
- * NestedForDepth
+ * PackageDeclaration
  * </li>
  * <li>
- * NestedIfDepth
+ * Regexp
  * </li>
  * <li>
- * NestedTryDepth
+ * RegexpSinglelineJava
  * </li>
  * <li>
- * NoWhitespaceAfter
+ * SuppressWarningsHolder
  * </li>
  * <li>
- * NoWhitespaceBefore
+ * TodoComment
  * </li>
  * <li>
- * NPathComplexity
+ * TrailingComment
  * </li>
  * <li>
- * OneStatementPerLine
+ * UncommentedMain
  * </li>
  * <li>
- * OperatorWrap
+ * UnnecessaryParentheses
  * </li>
  * <li>
- * OuterTypeNumber
+ * VariableDeclarationUsageDistance
  * </li>
  * <li>
- * PackageName
- * </li>
- * <li>
- * ParameterAssignment
- * </li>
- * <li>
- * ParameterName
- * </li>
- * <li>
- * ParameterNumber
- * </li>
- * <li>
- * ParenPad
- * </li>
- * <li>
- * RedundantImport
- * </li>
- * <li>
- * RedundantModifier
- * </li>
- * <li>
- * RequireThis
- * </li>
- * <li>
- * ReturnCount
- * </li>
- * <li>
- * RightCurly
- * </li>
- * <li>
- * SeparatorWrap
- * </li>
- * <li>
- * SimplifyBooleanExpression
- * </li>
- * <li>
- * SimplifyBooleanReturn
- * </li>
- * <li>
- * SingleSpaceSeparator
- * </li>
- * <li>
- * StaticVariableName
- * </li>
- * <li>
- * StringLiteralEquality
- * </li>
- * <li>
- * SuperClone
- * </li>
- * <li>
- * SuperFinalize
- * </li>
- * <li>
- * SuppressWarnings
- * </li>
- * <li>
- * ThrowsCount
- * </li>
- * <li>
- * TypecastParenPad
+ * WriteTag
  * </li>
+ * </ul>
+ * <p>
+ * Also, the filter does not support Javadoc checks:
+ * </p>
+ * <ul id="javadocChecks">
```

Comment:
this id should be unique in page please add some prefix in front  like "suppresionxpathfilter_"

Reply:
done